shared/chat/inbox/row/small-team/bottom-line.js

```diff
@@ -86,32 +111,42 @@ class BottomLine extends PureComponent<Props> {
       <Box
         style={{
           ...globalStyles.flexBoxRow,
+          alignItems: 'center',
           backgroundColor: isMobile ? backgroundColor : undefined,
-          flexGrow: 1,
-          maxHeight: height,
-          minHeight: height,
-          position: 'relative',
+          width: '100%',
+          flexShrink: 0,
+          height,
         }}
       >
-        <Box
-          style={{
-            ...globalStyles.flexBoxRow,
-            alignItems: 'flex-start',
-            bottom: 0,
-            justifyContent: 'flex-start',
-            left: 0,
-            position: 'absolute',
-            right: 0,
-            top: 0,
-          }}
-        >
-          {content}
+        {hasResetUsers && <Meta title="RESET" style={resetStyle} />}
+        <Box style={{flexGrow: 1, position: 'relative', height: '100%'}}>
+          <Box
+            style={{
+              ...globalStyles.flexBoxRow,
+              alignItems: 'flex-start',
+              bottom: 0,
+              justifyContent: 'flex-start',
+              left: 0,
+              position: 'absolute',
+              right: 0,
+              top: 0,
+            }}
+          >
+            {content}
+          </Box>
         </Box>
       </Box>
     )
   }
 }
 
+const resetStyle = {
```

Comment:
I think you need a `marginTop` on mobile 
![80f6baff-60a0-471a-9c4c-3ae7e1ffa843](https://user-images.githubusercontent.com/11968340/34166674-8247d7e2-e4ad-11e7-82ed-b1dfa6c72bd8.jpg)